flask/src/main.py

```diff
@@ -96,6 +99,20 @@ def __init__(self, import_name, *args, **kwargs):
 app = MyFlask(__name__)
 CORS(app)
 
+cache_config = {
+    "DEBUG": True,          # some Flask specific configs
+    "CACHE_TYPE": "RedisCache",  # Flask-Caching related configs
+    "CACHE_DEFAULT_TIMEOUT": 300
+}
+
+redis_host = os.environ.get("REDISHOST", "localhost")
```

Comment:
fallbacks mask bugs and missing configuration.
IMO these values should be explicitly set in `local.env` if they are not set it should fail

Reply:
sg, i'm fine with that.